CIP-1694/README.md

```diff
@@ -731,7 +746,7 @@ The **governance group** consists of all the new protocol parameters that are in
 * governance voting thresholds ($P_1$, $P_{2a}$, $P_{2b}$, $P_3$, $P_4$, $P_{5a}$, $P_{5b}$, $P_{5c}$, $P_6$, $P_7$, $Q_1$, $Q_{2b}$, $Q_4$)
 * constitutional committee term limits
 * governance action expiration
-* governance action deposit (`govDeposit`)
```

Comment:
Are governance action deposits being removed? because they are still referenced many times in this proposal

Reply:
Nice catch, thanks. Yes, in this proposal, fees supersedes the deposit.

Reply:
I love this governance fee new protocol parameter idea. May I suggest to keep the `govDeposit` with the `minGovernanceFee` which would be small at first and both ajustable through governance actions (protocol params change). So you get the best of both worlds. 

- A small `minGovernanceFee` shared amongs the DReps, so you keep the incentives of submitting goverance actions.
and
- A bigger `govDeposit` to mitigate the governance action load.

Also a small fix suggestion on Rational links:
![image](https://github.com/cardano-foundation/CIPs/assets/112007661/94c026aa-073d-4e21-a531-6411636615a7)